lms/static/sass/course/courseware/_courseware.scss

```diff
@@ -92,6 +92,93 @@ html.video-fullscreen {
       display: inline;
     }
   }
+
```

Comment:
This block of sass is now too broadly defined and will unexpectedly affect other areas of the app. Also, it is using hardcoded color values, and bits that should be using extends instead of being hardcoded. At a minimum, it should be inside a ".exam-timer" wrapper, but should just use existing button styles found in lms/static/sass/elements/_controls.scss